### ISSUE TITLE
Expose common esentutl maintenance tasks in Management UI

### ISSUE THREAD

**Issue by ramonsmits** [Particular/ServiceControl#957]
Our [compacting](https://docs.particular.net/servicecontrol/db-compaction) and [troubleshooting](https://docs.particular.net/servicecontrol/troubleshooting) doco make use of the following tasks:

Repair: `esentutl /p Data`
Recovery: `esentutl /r RVN /l logs /s system`
Defrag: `esentutl /d Data`

Having easy access to these tasks makes maintenance much easier.

**Comment by WojcikMike** [comment]
I believe that: #2311 is a subset of this issue

**Comment by ramonsmits** [comment]
@WojcikMike No, different issues. This is about maintenance tasks related to the database, the other issue is about repairing the "windows service" and queue setup.

**Comment by soujay** [comment]
Here is a case that could have benefitted with this functionality
[Case : 00080893](https://particular-software.lightning.force.com/lightning/r/Case/5004G00002ZZuupQAD/view)
